### PR TITLE
chore: update sdk and cli versions

### PR DESCRIPTION
**Pull request by sbckr**

### CODE REVIEW

File: docs/getting-started/platform-setup.md [hunk 1]
```diff
@@ -321,10 +321,18 @@ The public IP eventually appears in column `EXTERNAL-IP`.
       config:
          domain:
             ${EXTERNAL_IP}.sslip.io: ""
+         defaults:
+            max-revision-timeout-seconds: "36000"
     EOF
     kubectl apply -f knative-serving.yaml
     ```
 
+    The configuration above will also increase knative's default
```
Comment (strieflin) [inline docs/getting-started/platform-setup.md:330]: It's Knative not knative

File: docs/getting-started/platform-setup.md [hunk 2]
```diff
@@ -321,10 +321,18 @@ The public IP eventually appears in column `EXTERNAL-IP`.
...
+    [max-revision-timeout-seconds](https://knative.dev/v1.9-docs/serving/configuration/config-defaults/#revision-timeout-seconds)
+    from `600` to `36000` seconds (10h). This is required as ephemeral
+    computations are executed as -called knative activations and are therefore
```
Comment (strieflin) [inline docs/getting-started/platform-setup.md:333]: "-called" -> "so-called", "knative" -> "Knative"

**Review by strieflin — CHANGES_REQUESTED** [review]

File: docs/getting-started/platform-setup.md (hunk 2 above)
Comment (sbckr) [reply docs/getting-started/platform-setup.md]: fixed

File: docs/getting-started/platform-setup.md (hunk 1 above)
Comment (sbckr) [reply docs/getting-started/platform-setup.md]: fixed

**Review by strieflin — APPROVED** [review]
LGTM

**Pull request merged**